doc/tutorial/search/index.md

```diff
@@ -86,7 +86,7 @@ You are given the text
 dna4_vector text{"CGCTGTCTGAAGGATGAGTGTCAGCCAGTGTAACCCGATGAGCTACCCAGTAGTCGAACTGGGCCAGACAACCCGGCGCTAATGCACTCA"_dna4};
 \endcode
 Create a seqan3::fm_index over the reference, store the index and load the index into a new seqan3::fm_index object.
-Print whether the indices are identical or differ.
+Print whether the indices are identical or different.
```

Comment:
I guess this depends on how you read it:

* `are (identical or different)` -> (are identical) or (are different)
* `(are identical) or differ` -> (sth. are identical) or (sth. differ)